src/Tokenizer/Analyzer/ClassyAnalyzer.php

```diff
@@ -57,6 +57,10 @@ public function isClassyInvocation(Tokens $tokens, int $index): bool
             return true;
         }
 
+        if (\PHP_VERSION_ID >= 8_00_00 && $nextToken->equals(')') && $prevToken->equals('(') && $tokens[$tokens->getPrevMeaningfulToken($prev)]->isGivenKind(T_CATCH)) {
```

Comment:
will this work with https://wiki.php.net/rfc/multiple-catch ?

Reply:
I believe it won't work... 🫠

Reply:
I tested it locally and it works.

Reply:
Yeah, just tested it and it works, but it would be good to have that test case too. I thought ensuring prev `(` _and_ next `)` won't work because of `|`. It got me curious and I tested how it behaved before changes provided in this PR, and... it works too 🤔.

<img width="909" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/384618a0-5b0f-45b3-b29b-feef0a955000">

@fancyweb what was your failing case for this change? I believe we need improve `CONTRIBUTING.md` and make it explicit that every bug fix should start with failing test case, then code changes should be applied after CI failure. My bad I did not request for it here.

Reply:
FYI: I've improved contributing guide in #7204

Reply:
> what was your failing case for this change?

The first test I committed, a simple:
```php
use Exception;
try {
} catch (Exception) {
}
```
that was not correctly fixed.

Reply:
OK, got it. Here's what was fixed:

<img width="1209" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/31d55006-386b-4e33-9383-7720ecf42261">

Non-capturing `catch` with single type was not imported before, as PR's description states.